.github/workflows/release-pipeline.yaml

```diff
@@ -23,62 +30,31 @@ jobs:
       with:
         username: ${{ secrets.DOCKER_USERNAME }}
         password: ${{ secrets.DOCKER_PASSWORD }}
-    - name: '[alpine] Docker metainformations'
-      id: docker_meta_alpine
-      uses: crazy-max/ghaction-docker-meta@v3
-      with:
-        images: metacontrollerio/metacontroller # list of Docker images to use as base name for tags
-    - name: '[alpine] Build and push'
-      id: docker_build_alpine
-      uses: docker/build-push-action@v2
+    - name: Login to github registry
+      if: github.event_name != 'pull_request'
+      uses: docker/login-action@v1
       with:
-        context: .
-        file: ./Dockerfile
-        platforms: linux/amd64,linux/arm64,linux/arm
-        push: ${{ github.event_name != 'pull_request' }}
-        tags: ${{ steps.docker_meta_alpine.outputs.tags }}
-        labels: ${{ steps.docker_meta_alpine.outputs.labels }}
-        build-args: |
-          TAG=${{ env.GITHUB_REF_SLUG }}
-    - name: '[distroless] Docker metainformations'
+        registry: ghcr.io
```

Comment:
query: Is this a change in registry? What is the current registry?

Reply:
query: Should there be a doc update as well that mentions about the new registry?

Reply:
@AmitKumarDas re: registry -  logging to both, and the name of the image determines to which ones image will be pushed, as I defined both names in `images` - the push will save to both.

I will add change to documentation separately, it should be also visible on `Packages` page